### PR TITLE
Refs [TASK] [34908] [Create UI Home screen]

### PR DESCRIPTION
**Pull request by manhhung23399**
## Related Tickets
- [#Task 34908: Create UI Home screen](https://edu-redmine.sun-asterisk.vn/issues/34908)
## WHAT
- Create Viewpager for slide banner
- Create RecyclerView Trending Now
- Create RecyclerView Upcoming
- Create RecyclerView Now Playing
- Item Recycler, ViewPager
## Review Checklist

Category | View Point | Description | Expected Reviewer Answer | Self review | Reviewer2 (name)
--- | --- | --- | --- | --- | ---
Conventions | Does the code follow Sun* coding style and coding conventions? | https://github.com/framgia/coding-standards/tree/master/eng/android | YES |<li>- [ ] yes</li>|<li>- [ ] yes</li>  
Redmine | Does the ticket follow Sun* Redmine working process?  | https://github.com/framgia/Training-Guideline/blob/master/WorkingProcess/redmine/redmine.md| YES |<li>- [ ] yes</li>|<li>- [ ] yes</li>  
Documentation | Is there any incomplete code? If so, should it be removed or flagged with a suitable marker like ‘TODO’? |  | YES |<li>- [ ] yes</li>|<li>- [ ] yes</li>  
## Notes (Optional)
*(Impacted Areas in Application(List features, api, models or services that this PR will affect))*

*(List gem, library third party add new)*

*(Other notes)*

### CODE REVIEW

File: app/src/main/res/layout/item_slide.xml [hunk 1]
```diff
@@ -0,0 +1,43 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    android:layout_width="match_parent"
+    android:layout_height="@dimen/dp_220">
+
+    <ImageView
+        android:id="@+id/slideImageView"
```
Comment (huytq-2422) [inline app/src/main/res/layout/item_slide.xml]: xem lại conventions về cách đặt tên nhé

File: app/src/main/res/layout/item_slide.xml [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+        android:layout_width="0dp"
+        android:layout_height="0dp"
+        android:scaleType="centerCrop"
+        app:layout_constraintBottom_toBottomOf="parent"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintHorizontal_bias="0.0"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toTopOf="parent"
+        app:layout_constraintVertical_bias="0.0" />
```
Comment (huytq-2422) [inline app/src/main/res/layout/item_slide.xml]: bias để làm gì thế ?

Comment (manhhung23399) [reply app/src/main/res/layout/item_slide.xml]: khoảng các giữa các view ràng buộc ạ

File: app/src/main/res/layout/fragment_home.xml [hunk 3]
```diff
@@ -13,11 +13,26 @@
...
             android:layout_width="0dp"
             android:layout_height="@dimen/dp_220"
-            android:scaleType="fitXY"
+            android:scaleType="centerCrop"
             app:layout_constraintEnd_toEndOf="parent"
             app:layout_constraintStart_toStartOf="parent"
             app:layout_constraintTop_toTopOf="parent" />
 
+        <com.google.android.material.tabs.TabLayout
+            android:id="@+id/indicatorTabLayout"
```
Comment (huytq-2422) [inline app/src/main/res/layout/fragment_home.xml]:
```suggestion
            android:id="@+id/tabIndicator"
```

**Pull request merged**